vector/src/main/java/im/vector/app/features/roomdirectory/createroom/CreateRoomFragment.kt

```diff
@@ -223,16 +224,19 @@ class CreateRoomFragment @Inject constructor(
         views.waitingView.root.isVisible = async is Loading
         if (async is Success) {
             // Navigate to freshly created room
-            if (state.isSubSpace) {
-                navigator.switchToSpace(
-                        requireContext(),
-                        async(),
-                        Navigator.PostSwitchSpaceAction.None
-                )
-            } else {
-                navigator.openRoom(requireActivity(), async())
+            if (state.openAfterCreate) {
```

Comment:
Why is `openAfterCreate` in the state too? In the args is enough (I think)

Reply:
roomName, isSubspace (..) could be also. I just put it in state like others, why should it not be?

Reply:
Maybe because it is just used by the Fragment (I think). Anyway, it is fine like that.